scripts/crx3.proto

```diff
@@ -0,0 +1,62 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
```

Comment:
Adding a second FOSS license into this source code. Need to adapt Yocto recipe @Hollingsworth-martin-bl 

Can you please point me to the original source file @lars-kipferl-bl ?

Reply:
Will try to find the BSD license file for the crx3.proto file and incorporate it into yocto recipe. Would be added in new PR